YouVsVirus/Assets/Scenes/Levelgethome/PlayerHouse.cs

```diff
@@ -26,16 +46,19 @@ private void Start()
         UnshowPlayer();
         endlevel = LevelSettings.GetActiveEndLevelController();
         // activate end level controller
-
+        endlevel = LevelSettings.GetActiveEndLevelController().GetComponent<EndLevelControllerBase>();
```

Comment:
If we have this, the previous line is obsolete, is it not?

Reply:
mhmmm, strange. I will investigate.

Reply:
Maybe I did that ;-)

Reply:
xD